infra/Pos/Slotting/Util.hs

```diff
@@ -90,9 +72,30 @@ getSlotStartEmpatically
 getSlotStartEmpatically slot =
     getSlotStart slot >>= maybeThrow (SEUnknownSlotStart slot)
 
+-- | Get last epoch empatically, which means
+-- that function throws exception when last epoch is unknown.
+getPenultEpochSDEmpatically
+    :: (MonadSlotsData m, MonadThrow m)
+    => m EpochSlottingData
+getPenultEpochSDEmpatically = do
+    lastEpoch   <- getEpochLastIndex  
+    let penultEpoch = lastEpoch - 1  
+    getEpochSlottingData penultEpoch >>= maybeThrow (SEUnknownLastEpoch penultEpoch)
+
+-- | Get last epoch empatically, which means
+-- that function throws exception when last epoch is unknown.
```

Comment:
This function doesn't make sense, because the last epoch slotting data is always known.